build.rs

```diff
@@ -0,0 +1,93 @@
+#[macro_use] extern crate error_chain;
+extern crate curl;
```

Comment:
It's better not to use libraries that have external dependencies. 
Taking this into account, reqwest seems to fit better.